src/StandardError.js

```diff
@@ -6,25 +6,25 @@ const StandardError = {
 	// Output the StandardError Object, optionally filtered down to a particular domain
 	show(domain) {
```

Comment:
Unrelated to this PR, but, thoughts on changing this method to `list`? I thought `show` was better since it's actually an Object that is returned instead of an Array, but in action I feel like it's a weird discrepancy between this and the other `list<Keys|Errors>` functions.

Reply:
Yeah, that's a good point. I think `list` is pretty good but may be a little bit confusing since it maps them. I think `list` is better than `show`, but I'm not fully convinced it's the best choice.